src/packages/@ncigdc/components/Links/InternalLinkWithContext.js

```diff
@@ -57,8 +55,4 @@ const InternalLinkWithContext = ({
   </LocationSubscriber>
 );
 
-InternalLinkWithContext.defaultProps = {
-  mergeQuery: mq,
```

Comment:
Some version incompatibility was allowing this to fail silently. Bug surfaced with the handling of the query error.